nocrash.py

```diff
@@ -143,6 +148,38 @@ def error(message):
         sleep(5)
         count = 0
 
+    elif ecode == 20:
+        log('Pull in new updates beforeo we do `pip` upgrades.')
+        if 'windows' not in str(platform.platform()).lower():
+            git.checkout('deploy')
+            git.pull()
+        else:
+            warn('Not pulling updates; we are on Windows')
+
+        count = 0
+        crashcount = 0
+
+        log('Initiating automated `pip` upgrade calls within userspace.')
+        if is_venv():
+            command = "python3 -m pip install --upgrade -r "
+        else:
+            command = "python3 -m pip install --upgrade -user -r "
```

Comment:
You missed a hyphen

```suggestion
            command = "python3 -m pip install --upgrade --user -r "
```

Reply:
This was fixed in 7a476c6 and 5d69e9b because I caught it in my other code changes.